doc/nrf/includes/matter/interface/intro.txt

```diff
@@ -0,0 +1,10 @@
+This section describes the user interface that is available on the Development Kit in this |matter_type|.
+You can learn here about the LEDs and Buttons names used in this documentation and its mapping to the names visible on a development kit.
+
+.. include:: /includes/matter/interface/interface_table.txt
+
+Development Kit interface
+==========================
+
+This |matter_type| implements the following interface available on nRF Connect development kit.
```

Comment:
This is confusing. What is "the following interface". There is nothing that follows.